src/packages/sdk/scripts/patchJest.js

```diff
@@ -0,0 +1,17 @@
+const resolveFrom = require('resolve-from')
```

Comment:
This may not be needed, try removing it

Reply:
Doesn't seem possible due to this test https://github.com/prisma/prisma/pull/7152/checks?check_run_id=2692975438#step:7:577